packages/mwp-app-render/src/components/uxcapture/UXCaptureImageLoad.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from 'react';
+
+type Props = {
+	mark: string,
+	src: string,
+};
+
+/**
+ * Creates an image tag with provide props
+ * and a UXCapture `onLoad` handler
+ *
+ * @see example https://github.com/meetup/ux-capture#image-elements
+ */
+const UXCaptureImageLoad = (props: Props) => {
+	const { mark, src, ...other } = props;
+
+	// TODO: handle merging of onLoad prop
+
+	const onLoadCallback = () => {
```

Comment:
`onLoad` will fire with an `event` argument - you can type it with `ev: SyntheticEvent<*>`